.github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml

```diff
@@ -62,6 +62,8 @@ jobs:
           # auto generated GitHub token to allow accurately detecting a build vs a re-run build
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NEXT_PUBLIC_APP_ENV: test
+          # Pass in local authority details that will be checked in UI tests
+          COUNCIL_WEBSITE_HOMEPAGE_URL: ${{ secrets.COUNCIL_WEBSITE_HOMEPAGE_URL }}
```

Comment:
>Do they require the secret value?
I think they would need to use whatever value they expect the tests to check.
These would be used during the build stage when cypress is setting up its testing environment. 

> Could they use another value for the purposes of testing?
They could, but it would depend on the tests. 
I've added an example of using a cypress environment variable, which I think may help facilitate this.

In our case, the tests are testing for the real thing, so I've passed in the actual value as its environment variable.

Reply:
> Should the other newly added environment variables for configurable content also have equivalent tests?

👍🏾